docs/tech_primer.md

```diff
@@ -46,10 +53,12 @@ How data gets up into the cloud
     -   Bind & attach of proxied devices
     -   Encapsulation anti-pattern
 -   **Proxied**: A device that is not direct and is managed through a gateway.
+-   **External**: A device (on-prem or off-prem) consumes data from an external
+    source and publishes it to Cloud IoT core
```

Comment:
take out the "and publishes it to Cloud IoT core" part -- once you're doing external stuff there are other paths of ingress. But, I thikn the "External" moniker still still apples.